src/modules/service-catalog/submitServiceItemRequest.tsx

```diff
@@ -0,0 +1,50 @@
+import type { Field } from "../ticket-fields";
+import type { ServiceCatalogItem } from "./data-types/ServiceCatalogItem";
+
+export async function submitServiceItemRequest(
+  serviceCatalogItem: ServiceCatalogItem,
+  requestFields: Field[],
+  associatedLookupField: Field | null | undefined,
```

Comment:
What happens it the field is null? we should prevent this scenario otherwise the submission will not work

Reply:
I added check for associatedLookupField